browser/src/layer/tile/ScrollSection.ts

```diff
@@ -1050,6 +1050,10 @@ export class ScrollSection extends CanvasSectionObject {
 			this.performHorizontalScroll(delta[0]);
 		}
 	}
+
+	shouldDrawForTileBounds (tileBounds: cool.Bounds) : boolean {
+		return tileBounds === null;
```

Comment:
Are we sure this is right? I'd expect us to check if the scroll bar (?) or somesuch intersected with tileBounds, rather than just checking for null ?